app/styles/app.scss

```diff
@@ -401,6 +321,26 @@ td:hover {
     }
 }
 
+.index-titles h1, h3{
+    margin-top:30px;
```

Comment:
We have utility helpers for margins and paddings. Check out the osf-style: https://centerforopenscience.github.io/osf-style/style.html